client/cache.go

```diff
@@ -87,6 +90,7 @@ func NewObjectCache(discoveryClient *discovery.DiscoveryClient, options ObjectCa
 		cache:           &sync.Map{},
 		gvkToGVRCache:   &sync.Map{},
 		discoveryClient: discoveryClient,
+		options:         options,
```

Comment:
Question: Was this just missed (or unnecessary) previously?

Reply:
I think it was just missed. The other options aren't tested (and I think that's ok; they'd be annoying to test for a low payoff).